src/Sylius/Bundle/ApiBundle/Command/SendShopUserVerificationEmail.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+/**
+ * @experimental
+ * @psalm-immutable
+ */
+class SendShopUserVerificationEmail
+{
+    /** @var string */
+    public $shopUserEmail;
+
+    /** @var string */
+    public $locale;
+
+    /** @var string */
+    public $channelCode;
```

Comment:
As you have constructor and getters, these variables could be protected

Reply:
But all commands have `public` properties, so let's keep them consistent